l3kernel/l3keys.dtx

```diff
@@ -774,6 +774,36 @@
 %   is not processed in any way by \cs{keys_set:nn}.
 % \end{variable}
 %
+% \subsection{Expanding the values of keys}
+%
+% To allow the user to apply expansion of values when the key is set, rather
+% than when they are used, key names can be
```

Comment:
I still find the distinction with "use" strange. What does "use" mean? The *key* isn't used when the (internal) variable, which was set to the value given, is used, the *variable* is used then. So what is key usage?

Reply:
Suggestion on how to phrase this?

Reply:
Needs more context to work out the correct phrasing here.

Reply:
To allow the user to apply expansion of values when the key is set, rather
 than when the value is later  used, ...

I agree with @Skillmon that it is not the key that is used. It is the value that is used at a later stage.

Reply:
I'd just drop the "later used" part. For instance "To allow expansion control of values inside key=value lists" (or something along the lines, the first part of @FrankMittelbach's suggestion would be fine).